docs/digger/installation/digger-prereqs.asciidoc

```diff
@@ -1,22 +1,45 @@
 == Prerequisites
 
-A Digger server can be installed using the https://github.com/aerogear/digger-installer[Digger Installer tool].
+The main components (jenkins, android-sdk, nagios) of the digger project are deployed to an OpenShift Container Platform Installation. With these components installed digger has the capability to build Android Applications. The user has the option to configure (via the digger installer) a mac server to allow for iOS Application builds. This is an optional step and the Mac Server pre-requisites section below can be ignore if a Mac Server is not part of the installation.
+
+The digger tool can be installed using the https://github.com/aerogear/digger-installer[Digger Installer tool].
 
 Before running the Digger installer, please make sure the following prerequisites are met:
 
-=== Ansible 2.2 and higher
+=== General pre-requisites
+
+==== Ansible 2.2 or higher
+
+Digger Installer requires a version of Ansible 2.2 or higher on the machine that the Ansible playbook will be executed from. You can check the installed Ansible version by running `ansible --version`.
+Please check http://docs.ansible.com/ansible/intro_installation.html[Ansible installation instructions] for information on how to install.
+
+==== Outbound internet access
+
+Digger requires outbound internet access to perform npm installations, make sure that all relevant nodes/servers have outbound internet access.
+
+==== Proxy Configuration (optional)
+
+If your environments are on a network behind a proxy, this can be configured through the installer. Digger supports proxy without authentication. Digger does not currently support proxy with authentication.
+
+=== OpenShift pre-requisites
+
+==== OpenShift 3.3 or higher
+
+* An OpenShift Container Platform Installation, and its command line tool (https://github.com/openshift/origin/releases/tag/v1.3.1[oc]) installed locally.
+* The OpenShift cluster should have the following PVs (Persistent Volumes) available
+** 40GB for Jenkins
+** 10GB for Android-sdk
+** 1GB for Nagios
 
-Install Ansible locally. You can check the installed Ansible version by running `ansible --version`.
-Please check http://docs.ansible.com/ansible/intro_installation.html[Ansible installation instructions].
+==== Access to dockerhub
 
-=== Java 2.2 and higher
```

Comment:
Might be worth saying that java needs to be installed on the oscp master node and if its not we will do it